src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue15695.cs

```diff
@@ -0,0 +1,47 @@
+#if !MACCATALYST
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue15695 : _IssuesUITest
+{
+	public override string Issue => "Support for Switch OFF State color";
+
+	public Issue15695(TestDevice device) : base(device)
+	{
+	}
+
+	[Test]
+	[Category(UITestCategories.Switch)]
+	public void VerifySwitchOffColor()
+	{
+		App.WaitForElement("Switch");
+		VerifyScreenshot();
```

Comment:
Pending snapshots. Already available on the latest build.

Reply:
I've added snapshots for all platforms